.github/BOTMETA.yml

```diff
@@ -1585,6 +1589,7 @@ macros:
   team_aci: brunocalogero dagwieers devarshishah3 fadallar jmcgill298 koladiya mtorelli rsmeyers schunduri smnmtzgr
   team_aireos: jmighion
   team_aix: bcoca dagwieers d-little flynn1973 gforster kairoaraujo marvin-sinister mator molekuul MorrisA ramooncamacho wtcross
+  team_aruba: karthikeyan-dhandapani
```

Comment:
@karthikeyan-dhandapani Are you missing jmighion in this team ?

Reply:
Hi @Akasurde, he removed himself from the maintainer and assigned team_ansible with this pull request still open. #61469

Reply:
Got it, thanks for the information. Could you please in-corporate changes from 61469 in this PR ? I will mark that PR as duplicate of this after update.